ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/API.hs

```diff
@@ -311,15 +311,28 @@ data ChainDB m blk = ChainDB {
       -- Blocks unknown to the ChainDB will result in 'False'.
       --
       -- If the hash corresponds to a block that is known to be invalid, but
-      -- is now older than @k@, this function may return 'False'.
+      -- is now older than 'k', this function may return 'False'.
```

Comment:
I would leave this as `@k@`. `@` is for typewriter font (a la "`tt`"). `'` quotes are for top-level identifiers we expect to be in-scope. That's not the case here.